module/behaviour/planning/SimpleWalkPathPlanner/src/SimpleWalkPathPlanner.cpp

```diff
@@ -201,9 +201,9 @@ namespace behaviour {
 
                     arma::vec3 rBWw_temp = {ball.position[0], ball.position[1], fieldDescription.ball_radius};
                     rBWw                 = timeSinceBallSeen < search_timeout ? rBWw_temp :  // Place last seen
-                               Htw.i().x() + Htw.i().translation();          // In front of the robot
-                    arma::vec3 pos       = Htw.transformPoint(rBWw);
-                    position             = pos.rows(0, 1);
+                               Htw.i().x() + Htw.i().translation();                          // In front of the robot
+                    arma::vec3 pos = Htw.transformPoint(rBWw);
```

Comment:
:nauseated_face:

Reply:
Yeah this is what happened when I ran `./b format`. Then it failed again. Then I realised I haven't got the new docker image on here. Going through LLVM rn :laughing: Will fix it up when that's finished.

Reply:
I was about to message you asking if you'd updated your container :laughing: